source/javascripts/components/_Variable.js.erb

```diff
@@ -143,6 +146,22 @@ angular.module("BitriseWorkflowEditor").factory("Variable", function() {
 		return _.first(_.without(_.keys(angular.copy(variableConfig)), "opts"));
 	};
 
+	Variable.prototype.meta = function(newValue) {
+		return optionGetterSetter(this, "meta", newValue);
+	};
+
+	Variable.prototype.isExpose = function(newIsExposedValue) {
+		var newMetaValue;
+		if (newIsExposedValue !== undefined) {
+			newMetaValue = angular.copy(this.meta());
+			newMetaValue.is_expose = newIsExposedValue;
+		}
+
+		var newMeta = optionGetterSetter(this, "meta", newMetaValue);
```

Comment:
`newMeta` -> `meta`. Also, we should rather call `newIsExposedValue` -> `isExpose` and `newMetaValue` -> `meta`.